data/data/agent/systemd/units/agent-start-ui.service.template

```diff
@@ -0,0 +1,27 @@
+### This is currently a draft placeholder for running the UI, to be completed
+
+[Unit]
+Description=Service that runs the Agent Installer UI
+Wants=network-online.target assisted-service.service
+Conflicts=agent-register-cluster.service agent-import-cluster.service
+After=network-online.target assisted-service.service
+PartOf=assisted-service-pod.service
+ConditionPathExists=/etc/assisted/node0
+ConditionPathExists=/etc/assisted/interactive-ui
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=/usr/local/share/assisted-service/agent-images.env
+EnvironmentFile=/etc/assisted/rendezvous-host.env
+ExecStartPre=/bin/rm -f %t/%n.ctr-id
+ExecStartPre=/usr/local/bin/wait-for-assisted-service.sh
+ExecStart=podman run --net host --cidfile=%t/%n.ctr-id --cgroups=no-conmon --log-driver=journald --restart=on-failure:10 --pod-id-file=%t/assisted-service-pod.pod-id --replace --name=agent-installer-ui --env AIUI_APP_API_URL registry.ci.openshift.org/ocp/4.19:assisted-install-ui
```

Comment:
I think the assisted-installer-ui image URL should come from /usr/local/share/assisted-service/agent-images.env which is generated by get_container_images.sh?

Reply:
Until the assisted UI will be part of the release payload (there's a ticket for that https://issues.redhat.com/browse/AGENT-426), we'll have to manually include it in the ISO. The current value allows testing it, we'll change it in a followup PR after the [builder script](https://github.com/openshift/agent-installer-utils/pull/39) will manage it.